electron/app/main.js

```diff
@@ -445,6 +445,7 @@ const handleCommand = function(cmd) {
     }
     else if ('open' in cmd) {
         createWindow(cmd);
+        console.log('dfdffdf')
```

Comment:
tidy up

Reply:
thats weird i thought i got rid of them all.  good catch :)